lib/utils/object.js

```diff
@@ -6,6 +6,35 @@
  * all unused exports should be removed by tree-shaking.
  */
 
+// @link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
+if (typeof Object.assign != "function") {
+    Object.assign = function(target, varArgs) {
+        // .length of function is 2
+        "use strict"
+        if (target == null) {
+            // TypeError if undefined or null
+            throw new TypeError("Cannot convert undefined or null to object")
+        }
+
+        let to = Object(target)
+
+        for (let index = 1; index < arguments.length; index++) {
+            const nextSource = arguments[index]
+
+            if (nextSource != null) {
```

Comment:
If we turn on ESLint again, w'll probably need a lint comment to stop it from complaining about non strict inequality :stuck_out_tongue_closed_eyes: